packages/sol-compiler/src/utils/compiler.ts

```diff
@@ -152,18 +153,42 @@ export async function compileSolcJSAsync(
 
 /**
  * Compiles the contracts and prints errors/warnings
- * @param solcVersion Version of a solc compiler
+ * @param solidityVersion Solidity version
  * @param standardInput Solidity standard JSON input
  */
 export async function compileDockerAsync(
-    solcVersion: string,
+    solidityVersion: string,
     standardInput: solc.StandardInput,
 ): Promise<solc.StandardOutput> {
     const standardInputStr = JSON.stringify(standardInput, null, 2);
-    const dockerCommand = `docker run -i -a stdin -a stdout -a stderr ethereum/solc:${solcVersion} solc --standard-json`;
-    const standardOutputStr = execSync(dockerCommand, { input: standardInputStr }).toString();
-    const compiled: solc.StandardOutput = JSON.parse(standardOutputStr);
-    return compiled;
+    const dockerArgs = [
+        'run',
+        '-i',
+        '-a',
```

Comment:
Optional nit: It could be clearer to group the options with their arguments like:

```
'-a stdin'
```

I'm pretty sure that's kosher (the docs for stream don't define a schema for the strings that can be elements of `args`), but please correct me if I'm wrong.

Reply:
Yeah the args have to be broken up. Otherwise it would be executed as `docker '-a stdin' ...`. I can group them on the same line and disable the prettier.

Reply:
I see, up to you. I'm fine with it as-is.